src/app/main/main.component.js

```diff
@@ -44,6 +46,10 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       url: "/thank-you.html",
       template: '<thank-you></thank-you>'
     })
+    .state('profile', {
+      url: "/profile.html",
```

Comment:
yes, profile page just adds another link to the top 'Payment Methods'. That is about it =) But it serves as a parent component for 'payment methods' and I assume it will be for manageGiving etc